velox/exec/tests/MergeJoinTest.cpp

```diff
@@ -592,6 +592,38 @@ TEST_F(MergeJoinTest, semiJoin) {
       core::JoinType::kRightSemiFilter);
 }
 
+TEST_F(MergeJoinTest, rightJoin) {
```

Comment:
can we also extend `testJoin()` above to test right outer joins (around line 156)? It would be nice to test right joins to compare with DuckDB, and to compare that left outer and right outer return the same results (if you flip the join sides).

Reply:
@pedroerp Yes. Updated.